wallet-new/src/Cardano/Wallet/Kernel/Wallets.hs

```diff
@@ -59,7 +59,8 @@ instance Exception CreateWalletError
 -------------------------------------------------------------------------------}
 
 -- | Creates a new HD 'Wallet'.
-createHdWallet :: PassiveWallet
+createHdWallet
+             :: PassiveWallet
```

Comment:
@k0001 As a rule of thumb, let's not change other functions only for style purposes unsolicitedly, please 😉